frontend/src/components/review/AddReview.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+    <div>
+        <div v-if="hasToken" class="add-review">
+            <b-field class="media">
+
+                <div class="media-left">
+                    <img
+                        src="https://ss0.4sqi.net/img/venuepage/v2/add_tip_blank_avatar@2x-4321684c656168f26ae9208901a9d83e.png"
+                        height="32"
+                        width="32"
+                    >
+                </div>
+
+                <b-field class="media-content">
+                    <div class="content-wrapper">
+                        <b-input v-model="newReview.description" maxlength="500" type="textarea" />
+
+                        <b-taglist>
+                            <b-tag 
+                                v-for="(file, index) in photos"
+                                :key="index"
+                                type="is-primary"
+                            >
+                                {{ file.name }}
+                                <button 
+                                    class="delete is-small"
+                                    type="button"
+                                    @click="deletePhoto(index)"
+                                />
+                            </b-tag>
+
+                            <b-upload 
+                                v-model="photos"
+                                accept="image/*"
+                                multiple
+                            >
+                                <span class="tag is-light">
+                                    <a>Add photo</a>
+                                </span>
+                            </b-upload>
+                        </b-taglist>
+                    </div>
+                </b-field>
+
+                <div class="media-right">
+                    <button
+                        class="button is-primary"
+                        @click="onAddReview"
+                    >Post</button>
+                </div>
+            </b-field>
+        </div>
+        <div v-else class="add-review media">
+            <div class="media-left">
+                <img
+                    src="https://ss0.4sqi.net/img/venuepage/v2/add_tip_blank_avatar@2x-4321684c656168f26ae9208901a9d83e.png"
+                    height="32"
+                    width="32"
+                >
+            </div>
+            <span class="media-content">
+                <a>Login</a>, to leave a review.
+            </span>
+            <div class="add-review-btn media-right">
+                <button
+                    class="button is-primary"
+                    disabled
+                >Post</button>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { mapActions } from 'vuex';
```

Comment:
you can use one import construction

`import { mapGetters, mapActions } from 'vuex';`